### DISCUSSION TITLE
Add support for being able to specify date ranges in exports

### DISCUSSION THREAD

**Discussion by PacificGilly** [django-import-export/django-import-export#1887] [Ideas]
Apologies if I've missed this in the documentation!

It would be a remarkable ability to control how much data you'd like to export by having the option to specify the date ranges in the admin UI you'd like. Being given the option for some standard time ranges (e.g. last 24 hours, last week, last month, etc) would also be helpful. At least having a calendar/time box for you to specify which datetime ranges to download would be awesome!

**Comment by matthewhegarty** [comment]
hi 
It's certainly possible to customize the export forms.  Details on how to get started are [here](https://django-import-export.readthedocs.io/en/latest/admin_integration.html#customize-admin-export-forms).

In the example, an author field is used to filter the export, so you can customize your form however you like to filter on date ranges.

**Reply by PacificGilly** [reply]
Ah thank you, I'll have a play around with this awesome extension and see if it works as expected 🙂
